cmd/triggerrun/cmd/root_test.go

```diff
@@ -53,8 +53,7 @@ func TestReadTrigger(t *testing.T) {
 		},
 		Spec: v1alpha1.TriggerSpec{
 			Bindings: []*v1alpha1.TriggerSpecBinding{
-				{Name: "triggerSpecBinding1"},
-				{Name: "triggerSpecBinding2"},
+				{Name: "triggerSpecBinding"},
```

Comment:
the name in the yaml file is "trigger-spec-binding" . We should update that here.
Same for the template.